### ISSUE TITLE
Add download site to RGP Lua

### ISSUE THREAD

**Issue by rpatters1** [finale-lua/jw-lua-scripts#123]
In keeping with Jari's original idea of integrating a script repository with JW Lua, it occurred to me that a simple link button in RGP Lua that launches to our documentation site would be a simple step in that direction. Which is the most "official" top-level URL?

**Comment by Nick-Mazuk** [comment]
For users, it would probably be: https://jwluascripts.com/help

For developers, it's: https://jwluascripts.com/docs/getting-started

Also, is there any reason why someone may wish to use JW Lua instead of RGP Lua? I should probably also update the docs to link to RGP Lua instead of JW Lua since Jari has given it his blessing.

**Comment by rpatters1** [comment]
As to why someone would wish to use JW Lua, anyone still running on a 32-bit version of Finale would have to use JW Lua. Also, the 64-bit version still works quite well in Windows. There are no issues with the edit dialog scrolling, so it's still a handy dev platform for quick prototyping at least.

Also, the Windows version is not facing imminent incompatibility with CPU chip changes. Although there is an ARM version of Windows, it is not necessarily the future of Windows, and I am not aware of any plans at Makemusic to support native operation with ARM Windows.

**Comment by rpatters1** [comment]
I should add that for Windows users it is perfectly feasible to continue running much older versions of Finale than it is on Mac. I'm fairly certain I could open Finale 2000 under the latest version of Win10. (I'll try after I finish migrating my Win dev env.) And for sure I can run Finale 2010 with JW Lua under the current version of Win 10.

**Comment by rpatters1** [comment]
I'm closing this, because we've addressed what I needed to know.

**Comment by Nick-Mazuk** [comment]
Ok, so if I understand you correctly, the reasons why someone would choose JW Lua over RGP Lua are:

1. If they're on a 32-bit version of Finale
2. If they wish to develop scripts on Windows for the developer experience

For all other cases, there's no real need to use JW Lua instead, though especially on Windows JW Lua works perfectly fine.

---

Is that roughly what you're saying?

For context, I'm trying to figure out if I should rewrite the docs to only point to RGP Lua as that's the most recent canonical version, or if I should also link to JW Lua.

**Comment by rpatters1** [comment]
That's all I can think of. Maybe also if they have scripts that make
extensive use of UserValueInput, but that's fairly easy to replace with
FCCustomWindow. (I've replaced it in all of ours.)

32-bit Finale is probably more prevalent on Windows than Mac. I was able to
run my copy of WinFin 2000 on the latest Win 10, so there is a lot less
incentive to upgrade Finale on Windows. That said, I believe the earliest
version JW Lua 0.54 supports is F2010. At any rate, I asked this on the
list and that was the earliest version anyone mentioned.

Perhaps the main focus in the docs should be on RGP Lua, but a brief
mention of JW Lua aimed at 32-bit Finale users might be helpful.
(Developers can take care of themselves or use 3rd party tools as we do.)
